src/Core/View.js

```diff
@@ -832,8 +832,9 @@ class View extends THREE.EventDispatcher {
     }
 
     /**
-     * Searches for {@link Feature} in {@link ColorLayer}, under the mouse of at
-     * a specified coordinates, in this view.
+     * Searches for {@link FeatureGeometry} in {@link ColorLayer}, under the mouse of at
+     * a specified coordinates, in this view. Combining them per layer and in a Feature
+     * like format.
```

Comment:
small typos: 
`of` -> `or`
`a specified coordinates` -> `a specified coordinate`